dotCMS/src/main/java/com/dotcms/ai/api/DotAIAPIFacadeImpl.java

```diff
@@ -3,10 +3,9 @@
 import com.dotcms.ai.app.AppConfig;
 import com.dotmarketing.beans.Host;
 import com.dotmarketing.util.Logger;
-import io.vavr.Lazy;
+import org.bouncycastle.util.Arrays;
```

Comment:
I don't think we want to depend on this lib here. Let's find something else.

Reply:
Auto import.